packages/components/datetime/DateTime.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'DateTime'
+type: 'component'
+status: 'stable'
+slug: /components/datetime/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/datetime'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-datetime--basic'
+typescript: ./src/DateTime.tsx,./src/RelativeDateTime.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+In this package, there are two components (`DateTime` and `RelativeDateTime`) that will return a date following [Forma 36 guidelines](https://f36.contentful.com/guidelines/date-and-time/).
+
+## DateTime
+
+The DateTime component will format any date to a human friendly format and wrap it in a `<time>` tag.
+It allows four different forms of display that can be controlled by changing the `format` prop:
+
+| `format` option | Result                 |
+| --------------- | ---------------------- |
+| `full`          | 12 Aug 2020 at 8:00 AM |
+| `day`           | 12 Aug 2020            |
+| `time`          | 8:00 AM                |
+| `weekday`       | Wed, 12 Aug            |
+
+### Examples
+
+```tsx
+import { DateTime } from '@contentful/f36-datetime';
```

Comment:
you can remove the `import` statement line the `codeblock`, otherwise it throws an error, and don't render the component.